src/components/Facility/EncounterCard.tsx

```diff
@@ -20,88 +19,104 @@ export const EncounterCard = (props: EncounterCardProps) => {
   const { encounter } = props;
   return (
     <>
-      <div className="pb-16 block relative border-l-2 px-4 border-l-secondary-300 hover:border-primary-500 transition-all before:absolute before:-left-[7px] before:top-0 before:w-3 before:aspect-square before:bg-secondary-400 before:rounded-full hover:before:bg-primary-500 before:transition-all">
-        <Badge
-          variant={
-            completedEncounterStatus.includes(encounter.status)
-              ? "outline"
-              : "secondary"
-          }
-          className="inline-flex items-center gap-2 -translate-y-2"
-        >
+      <div className="pb-6 block relative border-l-2 px-4 border-l-secondary-300">
+        <div className="absolute -left-[12px] top-0 bg-white">
```

Comment:
Rm the bg-white; you will probably have to adjust the positioning since border is in parent div (maybe move it to a container div wrapping the card instead?).

Also switch to rem.